playground/src/experiments/experiments.js

```diff
@@ -0,0 +1,143 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import './experiments.scss';
+import template from './experiments.hbs';
+
+import events from '../events/events.js';
+import * as Button from '../button/button.js';
+import FlyIn from '../fly-in/fly-in.js';
+import * as ExperimentItem from './experiment-list-item.js';
+import createInput from '../input-bar/input-bar.js';
+
+const EXPERIMENTS_CONFIG_SOURCE_PATH =
+  'https://raw.githubusercontent.com/ampproject/amphtml/master/tools/experiments/experiments-config.js';
+const EXPERIMENTS_ID_PATTERN = /(?:id: ')(.*?)(?:')/gm;
+
+export const EVENT_TOGGLE_EXPERIMENT = 'event-toggle-experiment';
+export const EVENT_REMOVE_EXPERIMENT = 'event-remove-experiment';
```

Comment:
I'd move this to `ExperimentItem` because I think every event should live with the class that actually publishes it.